### DISCUSSION TITLE
problem instalation RVloader, original MAC lost

### DISCUSSION THREAD

**Discussion by ghost** [Aurelio92/RVLoader#79] [General]
Hi, I don't know if what I am going to write belongs here.  I tried to install RVloader on my Wii 4.3U, but at one point the installation failed and stopped giving image on the screen (I use HDMI), then I changed tv to use the component input and I can display again. 

Then I try to return to factory defaults, I succeed. I proceed to create the letterbomb, but the site tells me that it only works with wii devices. Then I realize that the installation of RVloader modified my MAC. Since then I have not been able to reinstall the homebrew and I don't know how to fix it. I have not wanted to use bootmii IOS with my NAND backup because it is known that it might go wrong.

Is there any way to fix this?

Thanks and best regards.

**Comment by Aurelio92** [comment]
Did you enable VGA? If so, it's normal that component/Wii2HDMI stops working, because VGA replaces it. You can enforce component by unplugging the cable and replugging it.

The MAC address got changed because you patched out WiFi, so the WiFi module (and its MAC address) is bypassed. It should still work with the letterbomb installer though
